apps/prairielearn/src/schemas/infoQuestion.ts

```diff
@@ -97,6 +99,12 @@ export const WorkspaceOptionsJsonSchema = z
   .strict()
   .describe('Options for workspace questions.');
 
+export const defaultWorkspaceOptions = WorkspaceOptionsJsonSchema.extend({
```

Comment:
I did it like this for the following reasons:

- We can use this 'default object' as a fallback for what to send to the database if no workspace is set. This lets us remove the fallback values from the `fromDisk/*.ts` stuff
- We can keep the required properties on the object